packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -50,84 +52,76 @@ function SelectionTool(props: Props) {
     disabled,
   });
 
+  const computeSelection = useCallback(
```

Comment:
This code was shared between `onPointerMove` and `onPointerUp`, so I extracted it.